common/djangoapps/student/views/dashboard.py

```diff
@@ -811,6 +813,8 @@ def student_dashboard(request):
             enr for enr in course_enrollments if entitlement.enrollment_course_run.course_id != enr.course_id
         ]
 
+    course_enrollments = Paginator(course_enrollments, getattr(settings, 'ITEMS_VISIBLE_DASHBOARD', 10)).page(page)
```

Comment:
Hi @huynguyen-groove ,
Can you please update this varibable ITEMS_VISIBLE_DASHBOARD name to `COURSE_ITEMS_ON_LMS_DASHBOARD`